facefusion/audio.py

```diff
@@ -0,0 +1,71 @@
+from typing import Optional, Any, List
+
+import numpy
+import scipy
+from functools import lru_cache
+from facefusion.ffmpeg import read_audio_buffer
+from facefusion.typing import Fps
+
+
+def monoize_audio(audio : numpy.ndarray[Any, Any]) -> numpy.ndarray[Any, Any]:
+	if audio.ndim > 1:
+		audio = numpy.mean(audio, axis = 1)
+	return audio
+
+
+def cleanse_audio(audio : numpy.ndarray[Any, Any]) -> numpy.ndarray[Any, Any]:
```

Comment:
cleanse? or clean_audio? what do you clean it from?

numpy.ndarray[Any, Any]) -> numpy.ndarray[Any, Any]:

please create alias types that give a better audio what is being processed